Okta.Xamarin/Okta.Xamarin/Services/IDataStore.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Okta.Xamarin.Services
+{
+    public interface IDataStore<T>
+    {
+        Task<bool> AddItemAsync(T item);
```

Comment:
We document public methods.

Reply:
This file is part of the Xamarin Forms template project and is used by the base application shell.  It is expected that this file will be modified or removed for the purposes of the consumer of the project.